sphinx/source/docs/user_guide/quickstart.rst

```diff
@@ -39,8 +39,8 @@ installation effort on all platforms and configurations, including Windows,
 and also installs the examples into the ``examples/`` subdirectory of your
 Anaconda or Miniconda installation directory.
 
-If you are confident that you have installed all needed dependencies such as
-NumPy you may instead use ``pip`` at the command line:
+If you are confident that you have installed all needed dependencies, such as
+NumPy, you may instead use ``pip`` at the command line:
```

Comment:
"such as NumPy" is a parenthetical element, so you'll want to put commas around it.